sdk/storage/azqueue/queue_client.go

```diff
@@ -21,6 +21,9 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/storage/azqueue/sas"
 )
 
+// ClientOptions contains the optional parameters when creating a Client.
+type ClientOptions base.ClientOptions
```

Comment:
We usually re-export these as a type alias, `type ClientOptions = base.ClientOptions`

Reply:
agreed, and changed.